src/model/queries/review.js

```diff
@@ -36,8 +36,20 @@ const getReviewsByBusiness = businessId => new Promise((resolve, reject) => {
   });
 });
 
+const setReview = (idPerson, idBusiness, content, evaluation) => new Promise((resolve, reject) => {
+  const sql = {
+    text: 'INSERT INTO review (person_id, business_id, content, evaluation) VALUES ($1,$2,$3,$4)',
+    values: [parseInt(idPerson), parseInt(idBusiness), content, parseInt(evaluation)],
```

Comment:
You have to add the radix parameter here. ParseInt takes 2 parameters, one is the number, and the second is to tell the function which base it will use to parse that number (decimal, binary, hex...etc). 

read [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) for more explanation.